README.md

```diff
@@ -77,6 +90,29 @@ function redirect () {
     window.location = location
   })
 }
+
+function register() {
+  // need to grab the code provided by GitHub
+  var code = window.location.href.match(/\?code=(.*)/)[1]
+  var body = {
+    code: code
+  }
+  var opts = {
+    // or /login to verify user info
+    uri: '/register',
+    body: body,
+    json: true,
```

Comment:
I believe we can set the data to be the value of `json` and then strip the headers + body value. I know we tried this and we scratched our heads but think it _is_ how it's supposed to work - would def make for a slightly more concise example I think (: